### PR TITLE
New Yorker should use the mobile markup instead of the responsive

### PR DESCRIPTION
**Pull request by isaactorres**
This:
- does not make any changes to the original source code found here: https://subscribe.newyorker.com/subscribe/newyorker/118423
- is updating `subscribe/newyorker/118423.html` to use the mobile markup found here when using a mobile device: https://subscribe.newyorker.com/subscribe/newyorker/118423
- is prettifying `subscribe/newyorker/118423.html`
- is adding a dev script to be able to make changes locally, while accessing the assets hosted on `https://subscribe.newyorker.com` using this dev script: `npm run dev:subscribe.newyorker.com`

### Original markup:
screen shot 2018-12-05 at 12 39 05 pm

### Local markup:
screen shot 2018-12-05 at 12 39 16 pm

To test:
1. `npm i && npm run dev:subscribe.newyorker.com`
2. Open two tabs in chrome using the smallest device in your responsive devices in chrome, for example: `iPhone 5/SE`
3. Open these two pages in the tabs:
- https://subscribe.newyorker.com/subscribe/newyorker/118423
- http://localhost:8000/subscribe/newyorker/118423

4. They should look identical. (There maybe some script errors that are not being handled by this PR)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "description": "hosted order forms on subscribe.[brand].com",
   "main": "index.js",
   "scripts": {
+    "dev:subscribe.newyorker.com": "http-server -p 8000 --proxy https://subscribe.newyorker.com",
```
Comment (thirtysixthspan) [inline package.json]: This is useful! Can we document it in the Readme?

**Comment by thirtysixthspan** [comment]
It's not clear to me. Did you download the form from https://subscribe.newyorker.com/subscribe/newyorker/118423 and call it newyorker.html? If so, I think it should instead be stored in a file at /subscribe/newyorker/118423.

**Comment by isaactorres** [comment]
> It's not clear to me. Did you download the form from https://subscribe.newyorker.com/subscribe/newyorker/118423 and call it newyorker.html? If so, I think it should instead be stored in a file at /subscribe/newyorker/118423.

Yep! Ok, I'll make those changes now

**Comment by isaactorres** [comment]
@thirtysixthspan Updated

File: package.json (hunk 1 above)
Comment (crccheck) [reply package.json]: I consider npm scripts to be self documenting, and don't put them in the README. I've found way too many repos where we documented them in the README and then changed the npm scripts without updating the README.

Comment (isaactorres) [reply package.json]: I'm fine either way. I agree that documentation can quickly become outdated if not maintained, but it is useful for on boarding new users.

**Pull request merged**